hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2788,6 +2778,18 @@
     // RV_CORE_IBEX //
     //////////////////
 
+    {
+      name: chip_sw_nmi_irq
+      desc: '''Verify the NMI interrupt to the CPU and correctness of the cause.
+
+            Trigger an NMI interrupt by the following two methods:
```

Comment:
it would be good to have both of these done in one test (and not two separate tests).
That way we can make sure the hardware for clearing the NMI such that the test can move on would be good too.  

This would be a good place to test the chip alert escalation clearing too.

Reply:
I think both, since in the event of an escalation NMI, SW would have to make sure the alert handler stops escalating (if the class is clearable), and then clear the NMI cause register.

Reply:
Good point @tjaychen!